sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "        code_configuration=CodeConfiguration(\n",
+    "            code=f\"{scoring_src_dir}\", scoring_script=\"score.py\"\n",
+    "        ),\n",
+    "        environment=Environment(\n",
+    "            conda_file=f\"{scoring_src_dir}/conda.yaml\",\n",
+    "            image=\"mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:latest\",\n",
+    "        ),\n",
+    "        instance_type=compute_sku_for_safety_proxy,\n",
+    "        instance_count=compute_count,\n",
+    "        environment_variables={\n",
+    "            env_key_of_uai_id: uai_client_id,\n",
+    "            env_key_of_aacs_endpoint: aacs_endpoint,\n",
+    "            env_key_of_llama_score_uri: llama_score_uri,\n",
+    "            env_key_of_subscription_id: subscription_id,\n",
+    "            env_key_of_resource_group_name: resource_group_name,\n",
+    "            env_key_of_workspace_name: workspace_name\n",
+    "        },\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "endpoint = ManagedOnlineEndpoint(\n",
+    "        name=endpoint_name,\n",
+    "        auth_mode=\"key\",\n",
+    "        identity=IdentityConfiguration(\n",
+    "            type=\"user_assigned\",\n",
+    "            user_assigned_identities=[\n",
+    "                ManagedIdentityConfiguration(resource_id=uai_id)\n",
+    "            ],\n",
+    "        ),\n",
+    "    )\n",
+    "# create online endpoint\n",
+    "ml_client.online_endpoints.begin_create_or_update(endpoint).result()\n",
+    "\n",
+    "endpoint = ml_client.online_endpoints.get(endpoint_name)\n",
+    "print(endpoint.identity.type)\n",
+    "print(endpoint.identity.user_assigned_identities)\n",
+    "\n",
+    "# create deployment\n",
+    "ml_client.online_deployments.begin_create_or_update(deployment).result()\n",
+    "# check status\n",
+    "deployment = ml_client.online_deployments.get(\n",
+    "        endpoint_name=endpoint_name, name=deployment.name\n",
+    "    )\n",
+    "print(deployment)\n",
+    "# Set traffic to 100% for deployment\n",
+    "endpoint.traffic = {str(deployment.name): 100}\n",
+    "ml_client.begin_create_or_update(endpoint).result()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 10. Test the Safety Enabled LLaMa online endpoint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 10.1 Prepare sample request"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "test_src_dir = \"./safety-llama-test\"\n",
+    "os.makedirs(test_src_dir, exist_ok=True)\n",
+    "print(f\"test script directory: {test_src_dir}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile {test_src_dir}/sample-request.json\n",
+    "{\"data\": \"Hello World\"}"
```

Comment:
We should have some "bad" examples to show the content safety moderation is happening as well

Reply:
Added harmful sample request.